src/components/Slider/Slider.js

```diff
@@ -0,0 +1,99 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styles from './Slider.module.scss'
+
+class SliderElement extends React.Component {
+    constructor(props) {
+        super(props)
+        this.itemRef = React.createRef();
+    }
+
+
+    static propTypes = {
```

Comment:
Is a best practice to leave propTyps at the top of the file above the class definition

Reply:
Changed for delivery 4